web/assets/css/layout.less

```diff
@@ -39,7 +39,7 @@ body,
   .scrollable-body {
     width: 100%;
     box-sizing: border-box;
-    padding: 10px;
```

Comment:
This looks awesome when scrolling large amounts of logs, just one note about this padding removal, it makes the build headers kinda go to the edge of the screen.

Padding removal:
<img width="525" alt="Screen Shot 2019-08-30 at 4 03 56 PM" src="https://user-images.githubusercontent.com/2362482/64048469-ccbfa280-cb3f-11e9-9435-c4b703d73783.png">


Master:
<img width="501" alt="Screen Shot 2019-08-30 at 3 59 39 PM" src="https://user-images.githubusercontent.com/2362482/64048384-9550f600-cb3f-11e9-9a8f-c672589048c1.png">

Looks like if you add double the margin as padding-right (20px), it might solve it! Let me know what you think!

Reply:
Good catch @jomsie. Unfortunately, I think that adding `padding-right` will push the scrollbar off the screen, like this:

![image](https://user-images.githubusercontent.com/6372810/64261477-9779d280-cefa-11e9-8dc6-629f51ffb5b6.png)

... and if we start using a `calc()` on the `width` property to fix that, we still get a messy gap where the scrollbar doesn't meet with the build list above it.

I chatted with @pivotal-jamie-klassen this morning about this, and we decided not to mess with the margin/padding for now. I've added a commit that puts `layout.less` back to normal, and results in this:

![v4w4H2Wv3g](https://user-images.githubusercontent.com/6372810/64261217-2b976a00-cefa-11e9-9265-2c2146867003.gif)

I think the visual impact of logs scrolling 'past' the sticky headers is minimal compared to the potential layout overhaul needed to solve the spacing in a way that doesn't impact scrollbars. We can look into the build page layout in order to clean up `position: fixed` elements and make it a little more flexible in the future, and make this look even cleaner at that point in time.